### ISSUE TITLE
Error in diffusion model?

### ISSUE THREAD

**Issue by hjalmarlucius** [zalandoresearch/pytorch-ts#102]
https://github.com/zalandoresearch/pytorch-ts/blob/62a18cfd0f19fc2247fc7e79e77f44b8dab6c4c9/pts/modules/gaussian_diffusion.py#L248

The loss function returns the aggregate mean scalar rather than the elementwise losses (https://pytorch.org/docs/stable/generated/torch.nn.functional.l1_loss.html). I think the TimeGrad model assumes it receives elementwise losses as it tries to weigh them.

**Comment by kashif** [comment]
hmm let me see...
